test/interval_tests/loop_tests.jl

```diff
@@ -57,17 +57,50 @@ function calc_pi3(N)
     sqrt(6*S3)
 end
 
+function calc_pi4(N)
+    S4 = @floatinterval(0)
+    II = @floatinterval(1)
+
+    for i in N:-1:1
+        S4 += II / (i^2)
+    end
+    S4 += II / @floatinterval(N, N+1)
+
+    sqrt(6*S4)
+end
+
+function calc_pi5(N)
+    S5 = @floatinterval(0)
+    II = @floatinterval(1)
+
+    for i in N:-1:1
+        S5 += 1 // (i^2)
+    end
+    S5 += 1 / @floatinterval(N, N+1)
+
+    sqrt(6*S5)
+end
+
 
 facts("Pi tests") do
 
+    big_pi = with_bigfloat_precision(256) do
+        big(pi)
+    end
+
```

Comment:
Actually in this case I don't think that's necessary, since all it's used for is to test that it lies inside the interval generated by the sums.